django_project/minisass_frontend/src/pages/Map/index.tsx

```diff
@@ -120,104 +126,127 @@ const MapPage: React.FC = () => {
-              >
-                Unmodified (NATURAL condition)
-              </Text>
-            </div>
-            <div className="flex flex-row gap-3 items-center justify-center w-auto">
-              <Img
-                className="h-6 w-7"
-                src={`${staticPath}img_alarm_green_400.svg`}
-                alt="alarm_One"
-              />
-              <Text
-                className="leading-[136.40%] max-w-[243px] md:max-w-full text-base text-black-900"
-                size="txtRalewayRomanRegular16"
-              >
-                Largely natural/few modifications (GOOD condition)
-              </Text>
-            </div>
-            <div className="flex flex-row gap-3 items-center justify-center w-auto">
-              <Img
-                className="h-6 w-7"
-                src={`${staticPath}img_alarm_orange_a200.svg`}
-                alt="alarm_Two"
-              />
-              <Text
-                className="text-base text-black-900 w-auto"
-                size="txtRalewayRomanRegular16"
-              >
-                Moderately modified (FAIR condition)
-              </Text>
-            </div>
-            <div className="flex flex-row gap-3 items-center justify-center w-auto">
-              <Img
-                className="h-6 w-7"
-                src={`${staticPath}img_twitter.svg`}
-                alt="twitter"
-              />
-              <Text
-                className="text-base text-black-900 w-auto"
-                size="txtRalewayRomanRegular16"
-              >
-                Largely modified (POOR condition)
-              </Text>
-            </div>
-            <div className="flex flex-row gap-3 items-center justify-center w-auto">
-              <Img
-                className="h-6 w-7"
-                src={`${staticPath}img_alarm_deep_purple_400.svg`}
-                alt="alarm_Three"
-              />
-              <Text
-                className="leading-[136.40%] max-w-[268px] md:max-w-full text-base text-black-900"
-                size="txtRalewayRomanRegular16"
-              >
-                Seriously/critically modified(VERY POOR condition)
-              </Text>
-            </div>
-            <div className="flex flex-row gap-3 items-center justify-center w-auto">
-              <Img
-                className="h-6 w-7"
-                src={`${staticPath}img_settings.svg`}
-                alt="settings"
-              />
-              <Text
-                className="text-base text-black-900 w-auto"
-                size="txtRalewayRomanRegular16"
-              >
-                No groups present
-              </Text>
-            </div>
-            <div className="flex flex-row gap-3 items-center justify-center w-auto">
-              <Img
-                className="h-6 w-7"
-                src={`${staticPath}img_arrowdown.svg`}
-                alt="arrowdown"
+          {/* Toggle button to show legend */}
```

Comment:
Can we rather use another image here, @amyburness Can suggest a nicer Image that represents a Legend.